nexus/src/db/model.rs

```diff
@@ -1584,14 +1645,14 @@ impl From<params::VpcSubnetUpdate> for VpcSubnetUpdate {
     }
 }
 
-impl_enum_type!(
+impl_enum_type2!(
     #[derive(SqlType, Debug)]
     #[postgres(type_name = "vpc_router_kind", type_schema = "public")]
     pub struct VpcRouterKindEnum;
 
-    #[derive(Clone, Debug, AsExpression, FromSqlRow)]
+    #[derive(Clone, Copy, Debug, AsExpression, FromSqlRow, PartialEq)]
     #[sql_type = "VpcRouterKindEnum"]
-    pub struct VpcRouterKind(pub external::VpcRouterKind);
+    pub enum VpcRouterKind;
```

Comment:
I do like the idea of this thing *actually* being an enum now; that's way more intuitive